src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -730,6 +775,15 @@ async ValueTask ProcessSettingsFrameAsync(long settingsPayloadLength)
                         case Http3SettingType.MaxHeaderListSize:
                             _maximumHeadersLength = (int)Math.Min(settingValue, int.MaxValue);
                             break;
+                        case Http3SettingType.EnableWebTransport:
+
+                            // Per https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3#section-3.1
+                            // an endpoint that receives a value other than "0" or "1" MUST close the
+                            // connection with the H3_SETTINGS_ERROR error code.
+                            if (settingValue != 0 && settingValue != 1)
+                                throw HttpProtocolException.CreateHttp3ConnectionException(Http3ErrorCode.SettingsError);
```

Comment:
Please add braces for all `if` statements
```suggestion
                            if (settingValue != 0 && settingValue != 1)
                            {
                                throw HttpProtocolException.CreateHttp3ConnectionException(Http3ErrorCode.SettingsError);
                            }
```